lib/languages.ts

```diff
@@ -293,6 +293,16 @@ const definitions: Record<LanguageKey, LanguageDefinition> = {
         formatter: null,
         previewFilter: null,
     },
+    hlsl: {
+        name: 'HLSL',
+        monaco: 'hlsl',
+        extensions: ['.hlsl', '.hlsli'],
+        alias: [],
+        logoUrl: 'dxc.png',
```

Comment:
Doesn't HLSL have a log that is not the one from the dxc? I think it would make more sense like that

Reply:
HLSL doesn't have a logo to my knowledge, and it seems like a logo is mandatory at the moment. I meant to ask about this actually because this image is just a placeholder. Do you have suggestions?

Reply:
Oh. I've fixed it in #3934, sorry about that. It might take a few days for it to land, but once it's there we should be ok just leaving this `null`

Reply:
Ah no worries, I'll look out for that PR then (also I realized I was commenting from the wrong github account, sorry about that, @jeremyong-az is indeed also me)